app/assets/v2/js/truncate-hash.js

```diff
@@ -44,50 +44,50 @@ new truncateHash();
  * ex:
  * <div data-metamask-address>1234567891233456789asd</div>
 */
-(function() {
-  this.getaddress = function(elem, _address) {
-    if (_address) {
-      if (elem.nodeName == 'INPUT') {
-        elem.value = _address;
-      } else {
-        elem.textContent = _address;
-        elem.setAttribute('title', _address);
-      }
-    } else {
-      web3.eth.getCoinbase(function(_, address) {
-        if (elem.nodeName == 'INPUT') {
-          elem.value = address;
-        } else {
-          elem.textContent = address;
-          elem.setAttribute('title', address);
-        }
-      });
-    }
-    new truncateHash();
-  };
+// (function() {
```

Comment:
all this commented out code makes me nervous that some functionality might be removed/ not suppprted anymore.. but i havent traced the codebase to see.